### DISCUSSION TITLE
MAKE FUNCTION NOT WORKING NEED HELO ON VS CODE

### DISCUSSION THREAD

**Discussion by Terpytoine** [microsoft/vscode-discussions#775] [Extension Development QnA] [answered]
make : The term 'make' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a 
path was included, verify that the path is correct and try again.
At line:1 char:1
+ make hello.c
+ ~~~~
    + CategoryInfo          : ObjectNotFound: (make:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : Co

[the remark this answers is not in the document]
**Reply by Terpytoine** [reply]
i cant use my make funtion in vs code trying to apply it

[the remark this answers is not in the document]
**Reply by gjsjohnmurray** [reply, marked as answer]
This forum is intended for developers who are creating VS Code extensions.

**Comment by olguzzar** [comment]
Closing as this question is off-topic for this forum, which is focused specifically on extension development.
